radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/application/LocalMempoolPeriodicSubmittor.java

```diff
@@ -57,11 +62,18 @@ private void act(RunningNetwork network, Command command, BFTNode node) {
 		return commands.observeOn(Schedulers.io());
 	}
 
-	public void run(RunningNetwork network) {
-		Observable.interval(1, 10, TimeUnit.SECONDS)
+	@Override
+	public void start(RunningNetwork network) {
+		commandsDisposable = Observable.interval(1, 10, TimeUnit.SECONDS)
```

Comment:
Would prefer to see this idempotent, so that starting an already started network didn't leak the disposable.